code/modules/clothing/factions/suns.dm

```diff
@@ -0,0 +1,572 @@
+	icon_state = "suns_lpkarmor"
+	item_state = "suns_pkarmor"
+	icon = 'icons/obj/clothing/faction/suns/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/suits.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/suit/toggle/suns/pkcoat
+	name = "peacekeeper coat"
+	desc = "An armored coat used during special occasions. This one is used in academic security."
+	icon_state = "suns_pkjacket"
+	item_state = "suns_overblack"
+	armor = list("melee" = 15, "bullet" = 30, "laser" = 10, "energy" = 10, "bomb" = 20, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 25)
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/suit/armor/vest/bulletproof/suns/captain
+	name = "decorated academic coat"
+	desc = "An armored coat intended for SUNS captains on the frontier. Go forth, and spread the message of the academy."
+	icon_state = "suns_captaincoat"
+	item_state = "suns_overblack"
+	icon = 'icons/obj/clothing/faction/suns/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/suits.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/suit/armor/vest/bulletproof/suns/xo
+	name = "academic staff coat"
+	desc = "A white coat used by SUNS academic staff. It designates the second in command on the ship."
+	icon_state = "suns_xojacket"
+	item_state = "suns_overwhite"
+	icon = 'icons/obj/clothing/faction/suns/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/suits.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+
+///////////////
+//Spacesuits//
+//////////////
+
+
+/obj/item/clothing/head/helmet/space/syndicate/suns
+	name = "SUNS space helmet"
+	icon_state = "suns_vachelm"
+	item_state = "suns_vachelm"
+	desc = "An academic standard spacesuit helmet. Normally reserved for low budget tasks in space."
+	icon = 'icons/obj/clothing/faction/suns/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/head.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/suit/space/syndicate/suns
+	name = "SUNS spacesuit"
+	icon_state = "suns_vacsuit"
+	item_state = "suns_vacsuit"
+	desc = "An academic standard spacesuit. Normally reserved for low budget tasks in space."
+	icon = 'icons/obj/clothing/faction/suns/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/suits.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+
+/////////
+//Hats//
+////////
+
+
+/obj/item/clothing/head/beret/hop/suns
```

Comment:
Presumably for the armor afforded. Could opt for Captain instead of HoP, but this beret in particular was created bespoke for the second in command role.